### ISSUE TITLE
Generated code is not formatted

### ISSUE THREAD

**Issue by PSeitz** [hyperium/tonic#868]
### Version

pascal@pascal-G533QM ~/L/D/q/c/quickwit-proto (conf_sort)> cargo tree | grep tonic
└── tonic v0.5.2
└── tonic-build v0.5.2

### Platform

Linux pascal-G533QM 5.12.0-051200-generic #202104252130 SMP Sun Apr 25 21:33:13 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

### Description

In our project quickwit https://github.com/quickwit-inc/quickwit, the code is not formatted or formatted differently when I run the build_script https://github.com/quickwit-inc/quickwit/blob/main/quickwit-proto/build.rs.

`cd  quickwit-proto;cargo build`

These are the generated files on my machine:
[cluster.txt](https://github.com/hyperium/tonic/files/7734471/cluster.txt)
[quickwit.txt](https://github.com/hyperium/tonic/files/7734472/quickwit.txt)

**Comment by davidpdrsn** [comment]
tonic-build's [rustfmt](https://github.com/hyperium/tonic/blob/master/tonic-build/Cargo.toml) feature should format the code.

**Comment by PSeitz** [comment]
Can I check somehow why it doesn't?

**Comment by davidpdrsn** [comment]
Read the code in tonic build 🤷‍♂️😂

Can you provide a reproduction script?

**Comment by PSeitz** [comment]
> Can you provide a reproduction script?

No, it happens only on my machine :)

**Comment by davidpdrsn** [comment]
Then I’m not sure what could be going on. Maybe something is wrong with your `rustfmt` installation 🤷‍♂️

**Comment by PSeitz** [comment]
The issue only occurs with the nightly compiler, stable works

**Comment by LucioFranco** [comment]
Does your nightly toolchain have rustfmt installed?

**Comment by PSeitz** [comment]
> Does your nightly toolchain have rustfmt installed?

Yes, e.g. this command completes `cargo +nightly fmt --all -- --check`

**Comment by LucioFranco** [comment]
Do you use `rustup override set nightly` in the directory?

**Comment by PSeitz** [comment]
I have no overrides 

```
rustup override list
     no overrides
```

Did you try to reproduce the issue?

formatting does not work (files changed)
```
git clone https://github.com/quickwit-inc/quickwit
cd quickwit;cd quickwit-proto;cargo +nightly build
```

formatting works (files unchanged)
``` 
git clone https://github.com/quickwit-inc/quickwit
cd quickwit;cd quickwit-proto;cargo +stable build
```
